### ISSUE TITLE
Icons misplaced

### ISSUE THREAD

**Issue by alaycock** [wyze/atom-seti-icons#65]
![image](https://cloud.githubusercontent.com/assets/894797/26126554/434b4ab6-3a43-11e7-95d8-df2a87d58329.png)
I just updated Atom to 1.17.0, which seems to have some style changes. I'm not sure if this is something that would be fixed in seti-icons, or here, but this is the package I'm using, so I figured I'd open it here.

**Comment by nilsborg** [comment]
Same here. Did this in my stylesheet:

```
.tab-bar {
  .tab {
    &, &.active {
      min-width: 10em;
    }

    .title {
      padding-left: 24px !important;
      text-align: left;

      &.title::before {
        margin-top: 7px;
        font-size: 22px;
        margin-left: -26px;
      }
    }
  }
}
```

**Comment by wyze** [comment]
Fixed in [v1.5.0](https://github.com/wyze/atom-seti-icons/releases/tag/v1.5.0).
